tests/api_connexion/endpoints/test_dag_endpoint.py

```diff
@@ -310,6 +310,7 @@ def test_should_respond_200(self, url_safe_serializer):
             "concurrency": 16,
             "dag_id": "test_dag",
             "dag_run_timeout": None,
+            "dataset_expression": {},
```

Comment:
`None` is probably a better default.  Would you agree @uranusjr ?

Reply:
Yeah None feels more natural, but either need to be handled by the client anyway, I don’t think it really matters.

Reply:
Changed it